lib/Parser/Parse.cpp

```diff
@@ -10473,8 +10473,16 @@ void Parser::FinishDeferredFunction(ParseNodePtr pnodeScopeList)
             pnodeFnc->sxFnc.pnodeVars = nullptr;
             m_ppnodeVar = &pnodeFnc->sxFnc.pnodeVars;
 
+            ParseNodePtr pnodeFncSaveNonLambda = nullptr;
+
+            pnodeFncSaveNonLambda = m_currentNodeNonLambdaFunc;
```

Comment:
m0currentNodeNonLamdaFunc should be null here - should you put the null assert here?